src/annotator/util/test/emitter-test.js

```diff
@@ -0,0 +1,53 @@
+import EventBus from '../emitter';
+
+describe('EvenBus', () => {
```

Comment:
```suggestion
describe('Emitter', () => {
```

Although these tests use `EventBus`, I think it would be more accurate to describe them as a set of tests for the `Emitter` class.